lib/npg_tracking/data/bait/find.pm

```diff
@@ -16,12 +16,19 @@ with qw/ npg_tracking::data::reference::find /;
 our $VERSION = '0';
 
 Readonly::Scalar my $STRAIN_ARRAY_INDEX => 1;
+Readonly::Scalar my $DEFAULT_BAIT_NAME => q{Standard};
+Readonly::Scalar my $RNA_LIBRARY_BAIT_NAME => q{Exome};
+
 
 has 'bait_name'     => ( isa => q{Maybe[Str]}, is => q{ro}, lazy_build => 1,
                         documentation => 'Bate name',);
 sub _build_bait_name {
   my $self = shift;
-  return $self->lims->bait_name;
+  my $bait_name = $self->lims->bait_name // $DEFAULT_BAIT_NAME;
```

Comment:
Using default on line 27 changes the previous behaviour of the code and can potentially break lots of code in the pipeline. Why is it needed?

Reply:
A better approach has been considered. Thanks for the input.